docs/toml-schema.md

```diff
@@ -49,6 +49,12 @@ members = [
     "rfcbot",
     "craterbot",
     "rust-timer",
+    # Any subset of members may hold custom roles, beyond "Team leader" which is
+    # controlled by the `leads` array above. See [[website.roles]]. Members with
+    # roles are written using an inline table as follows. A simple string member
+    # like "bors" is equivalent to {name = "bors", roles = []}.
```

Comment:
Can we mention that the roles listed with each member must also be present in the roles array?

Reply:
Done.